oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/RecoveryTest.java

```diff
@@ -157,6 +169,69 @@ public void recoverOther() throws Exception {
         assertThat(diff.deleted, containsInAnyOrder("/parent/test/c1"));
     }
 
+    @Test
+    @Ignore(value = "Causes OOM on travis and local as well for 1.8 branch")
```

Comment:
Why does this work on trunk and 1.22 branch but not on 1.8? Does it get less memory on 1.8 or need more?